Library/Formula/llvm.rb

```diff
@@ -84,6 +99,10 @@ def install
       fail "The Python bindings need the shared library."
```

Comment:
I'm guessing that this is an old message and the behaviour was previously different.